ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md

```diff
@@ -5725,6 +5729,10 @@ Set Link Aggregation Control Protocol (LACP) parameters.
                 activate: <bool>
                 route_map_in: <str>
                 route_map_out: <str>
+                next_hop:
```

Comment:
@ClausHolbechArista 
Let me know if you agree with the proposed schema before I continue.
I kept the same schema as we have on the peer group and validated options on the command line.
The following forms are accepted at the CLI:
```
router bgp 65000
   vrf test1
      address-family ipv4
         neighbor 10.255.251.3 next-hop address-family ipv6
   vrf test2
      address-family ipv4
         neighbor 10.255.251.3 next-hop address-family ipv6 originate
   vrf test3
      address-family ipv4
         no neighbor 10.255.251.3 next-hop address-family ipv6
```

Reply:
agreed. But I think we probably need to remove peer-groups from the vrf context completely (or at least most of it) since EOS does not accept that (it automatically moved the config to the global address family context).